docs/public-networks/reference/cli/options.md

```diff
@@ -1167,6 +1167,38 @@ The key-value storage to use. Use this option only if using a storage system pro
 
 For development use only, the `memory` option provides ephemeral storage for sync testing and debugging.
 
+### `--kzg-trusted-setup`
```

Comment:
Think this has come in from unrelated commit?

Reply:
Oops, yes. This should not have been included. Don't know how that happened since I created a separate topic branch for issue [Document new CLI option #1274](https://github.com/hyperledger/besu-docs/issues/1274).

Reply:
Removed the `kzg-trusted-setup` CL option from the topic branch since it was already merged into main in a previous issue.